src/Tests/Microsoft.Diagnostics.Monitoring.HostingStartup.UnitTests/ParameterCapturing/Pipeline/CaptureLimitPolicyProbesTests.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.Diagnostics.Monitoring.HostingStartup.ParameterCapturing.Pipeline;
+using Microsoft.Diagnostics.Monitoring.TestCommon;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Microsoft.Diagnostics.Monitoring.HostingStartup.UnitTests.ParameterCapturing.Pipeline
+{
+    [TargetFrameworkMonikerTrait(TargetFrameworkMonikerExtensions.CurrentTargetFrameworkMoniker)]
+    public class CaptureLimitPolicyProbesTests
+    {
+        [Fact]
+        public void EnterProbe_RequestStop_OnLimitReached()
+        {
+            // Arrange
+            TaskCompletionSource requestStop = new();
+            CaptureLimitPolicyProbes probes = new(new TestFunctionProbes(), captureLimit: 1, requestStop);
+
+            // Act
+            probes.EnterProbe(1, []);
```

Comment:
Probably should... (ahem) _capture_... the return value and assert the expected value.

Reply:
In general unit tests should only be testing one thing at a time so they should only fail for one reason. I can add another test to validate these probes pass through the uniqufier / arguments, but I'd rather not update the other tests to also validate that.